src/ActionList/Item.tsx

```diff
@@ -84,12 +84,21 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
       },
     }
 
+    const isTopLevelInteractive = () =>
```

Comment:
Is it worth checking for a `tabIndex`?

Reply:
It won't hurt. It's not strictly speaking an expected prop, but there are a few places in dotcom that pass it in.